internal/controller/workspace/workspace.go

```diff
@@ -103,9 +100,11 @@ func Setup(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter, poll, t
 	gcTmp := workdir.NewGarbageCollector(mgr.GetClient(), filepath.Join("tmp", tfDir), workdir.WithFs(fs), workdir.WithLogger(l))
 	go gcTmp.Run(context.TODO())
 
+	l.Info("Creating terraform connector")
```

Comment:
All of these newly added log lines seem more appropriate as debug logs to me - please switch them to `l.Debug` per https://github.com/crossplane/crossplane/blob/master/docs/contributing/observability_developer_guide.md#in-practice:

> Almost nothing is worth logging at info level. When deciding which logging level to use, consider a production deployment of Crossplane reconciling tens or hundreds of managed resources. If in doubt, pick debug. You can easily increase the log level later if it proves warranted.